src/extension/src/ActionHandler.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, logger, env_layer, telemetry_writer, utility, runtime_context
         self.stdout_file_mirror = None
         self.file_logger = None
         self.operation_id_substitute_for_all_actions_in_telemetry = str((datetime.datetime.utcnow()).strftime(Constants.UTC_DATETIME_FORMAT))
+        self.seq_no = self.ext_config_settings_handler.get_seq_no_from_env_var()
```

Comment:
- The class in init'd for every handler action (i.e. install, enable, reset, etc) since agent invokes the extension for each command. So the seq no remains same for one whole lifecycle of of the handler
- write_basic_status() executes a bit later in the code flow, during the setup for each handler action.
- Not a primary reason but an added advantage: writing tests was easier with seq no in init

First 2 were main reasons behind moving this to class init